test/unit-test/cellular_at_core_utest.c

```diff
@@ -903,11 +903,10 @@ void test_Cellular_ATStrtoi_Happy_Path( void )
 {
     CellularATError_t cellularStatus = CELLULAR_AT_SUCCESS;
     char * pStr = malloc( sizeof( char ) * ( strlen( CELLULAR_SAMPLE_STRTOL_HAPPY_CASE_STRING ) + 1 ) );
```

Comment:
malloc

Reply:
please refer to this [change](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/37/commits/d6ed9baaca207edc8cb5ab7fe890e2937975d2d8).